src/components/Question.jsx

```diff
@@ -120,25 +107,65 @@ export default class Question extends React.Component {
     this.setState({ showIcon: false });
   };
 
-  render() {
+  _getHeaderClassName = () => {
     const question = this.props.question;
-    const subQuestion = question[Constants.HAS_SUBQUESTION];
+    const collapsible = this.props.collapsible;
     const options = this.context.options;
-    const questionComponent = this.renderQuestion(question);
-    if (FormUtils.isHidden(question)) {
-      return null;
+    const subQuestion = question[Constants.HAS_SUBQUESTION];
+
+    let headerClassName = [];
+
+    headerClassName.push(
+      Question.getEmphasizedOnRelevantClass(question),
+      "question-header"
+    );
+
+    if (FormUtils.isWizardStep(question))
+      headerClassName.push("wizard-step bg-primary text-white");
+    if (FormUtils.isEmphasised(question))
+      headerClassName.push(Question.getEmphasizedClass(question));
+    if (FormUtils.isSection(question))
+      headerClassName.push("section-background");
+    if (collapsible) headerClassName.push("cursor-pointer");
+    if (
+      !FormUtils.isRelevant(question) &&
+      (options.debugMode ||
+        JsonLdObjectUtils.checkId(subQuestion, options.startingQuestionId))
+    ) {
+      headerClassName.push("show-irrelevant");
     }
+
+    return headerClassName;
+  };
+
+  render() {
+    const question = this.props.question;
+    const questionComponent = this.renderQuestion(question);
+    const subQuestions = question[Constants.HAS_SUBQUESTION];
+    let testedQuestion;
+
+    if (FormUtils.isTested(question)) testedQuestion = question;
     if (
```

Comment:
We agreed to use this instead of the condition + remove FormUtils.isTested
```
if ( isDebugged(question)) {}
```

```
isDebugged(question) {
   if (FormUtils.isRelevant(question)) return false;
   if (this.context.options.debugMode) return true;
   return JsonLdObjectUtils.checkId(
          [ question, ...getSubQuestions(question) ],
          this.context.options.startingQuestionId
        )
}
```

Reply:
Moved to #172